incydr/cli/cmds/directory_groups.py

```diff
@@ -39,6 +41,13 @@ def list_(ctx: Context, format_: TableFormat, name: Optional[str] = None):
         format_ = TableFormat.table
     client = ctx.obj()
     groups = client.directory_groups.v1.iter_all(name=name)
-    output_models_format(
-        groups, "Directory Groups", format_, None, client.settings.use_rich
-    )
+    if format_ == TableFormat.table:
```

Comment:
kinda random use of `if/else` between the two of these commands, not a big deal but would be cool if we could keep them all consistent imo